pkg/certificates/certs.go

```diff
@@ -170,31 +155,31 @@ func CreateControlPlaneCert(ctx context.Context, caKey *rsa.PrivateKey, caCertif
 }
 
 // CreateDataPlaneCert generates the certificate for the server
-func CreateDataPlaneCert(ctx context.Context, caKey *rsa.PrivateKey, caCertificate *x509.Certificate, expirationInterval time.Duration) (*KeyPair, error) {
+func CreateDataPlaneCert(ctx context.Context, caKey *rsa.PrivateKey, caCertificate *x509.Certificate, expirationInterval time.Duration, san string) (*KeyPair, error) {
```

Comment:
Ditto in terms of breaking signatures.

If `CreateControlPlaneCert` and `CreateDataPlaneCert` do the same thing now, it seems like a new function might be for the best long-run.

Reply:
Let's leave this for a subsequent PR. 
There are some opportunities to merge code and rethink what we name here as data-plane or control-plane and if it has anything to do with data or control. 
We also need to think if we need separation of data and control certificate wise...
If so, we don't do that today (considering that everything in knative-serving now uses the same certificate ;) )

Reply:
Now realizing that this relates to the comment below asking to fix now Public Naming...